setup.py

```diff
@@ -0,0 +1,10 @@
+from setuptools import setup, find_packages

```

Comment:
I will do the packaging myself in a future pull request.
Please remove this file from the PR.